### PR TITLE
Added post-mortem of Codecov security incident

### PR DESCRIPTION
**Pull request by sergei-maertens**
- [x] English version
- [x] Dutch version

### CODE REVIEW

File: src/pages/news/2021-04-16-codecov-security-update.en.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+## Codecov security update: Open Zaak is not affected
+
+On April 15th, we were informed by Codecov that they had a security incident that
+possibly affected Open Zaak repositories. We have checked our repositories and verified
+that everything is as expected.
+
+Unless you are using the `latest` image tag of Open Zaak
+or Open Notificaties Docker Hub images, **no action is required**. If you _do_ use the `latest` 
+tag in any of those images, our advise is to pull these images again. There is no indication
```
Comment (ericherman) [inline src/pages/news/2021-04-16-codecov-security-update.en.md]: I might write this as:

If you _do_ use the `latest` tag in any of those images, _and_ those images are old enough to no longer have build logs, then our advice is to pull these images again.

However, I'm also fine with it as-is (with s/our advise/our advice/).

**Review by ericherman — CHANGES_REQUESTED** [review]
Please change "our advise is" to "our advice is".

**Review by bvhme — APPROVED** [review]
Looks great. Compliments for the clear posts in English and Dutch!

PS, I'm a dyslect, so I might have not read spelling or grammar issues. However it looks good.

File: src/pages/news/2021-04-16-codecov-security-update.en.md (hunk 1 above)
Comment (sergei-maertens) [reply src/pages/news/2021-04-16-codecov-security-update.en.md]: oh, good catch! I like your version better, will change it :)

**Comment by Ainali** [comment]
Do we really want to back date the post to April 16 rather than putting in today's date?

**Comment by sergei-maertens** [comment]
> Do we really want to back date the post to April 16 rather than putting in today's date?

That's something I wasn't sure about either. I personally find it important to highlight we acted on the information from Codecov the same day we received this. I had hoped/expected to be able to publish the post sooner, but it stayed in internal review for a long time.

I think we can use the real publish dates instead of backdating, because the timeline of which action happened when is documented as well.

**Review by ericherman — APPROVED** [review]

**Comment by bvhme** [comment]
I recommend publishing it and figuring out the dating afterwards, possibly changing it in a separate update.

Secondly, I recommend using today's date. But longer ago is also OK.

**Comment by ericherman** [comment]
Not required, but might be nice to align the NL text change (if latest _and_ old) to match the EN version.

**Pull request merged**